src/main/features/claim/routes/fees.ts

```diff
@@ -8,43 +8,52 @@ import { RangeGroup } from 'app/fees/models/rangeGroup'
 import { FeesTableViewHelper } from 'claim/helpers/feesTableViewHelper'
 import { DraftClaim } from 'drafts/models/draftClaim'
 import { Draft } from '@hmcts/draft-store-client'
+import { ClaimAmountExceedsLimitError } from '../../../errors'
```

Comment:
Can you please change this to not be a relative import?

Reply:
fixed